test/NatickMallTest.js

```diff
@@ -0,0 +1,149 @@
+const assert = require("assert");
+const natickmall = require("./../site-scrapers/NatickMall");
+const nock = require("nock");
+const chai = require("chai");
+chai.use(require("chai-as-promised"));
+chai.use(require("chai-json"));
```

Comment:
you need to add this to package.json :)

Reply:
Good catch. Fixed. We actually don't need this.